install/third-party/blameless/install.yml

```diff
@@ -0,0 +1,14 @@
+id: third-party-blameless
+name: Blameless
+title: Blameless
+description: 
+  Site reliability engineering (SRE) platform with AI-driven incident resolution, SLOs/Error budgets, and reliability insights.  
+
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docsend.com/view/29zr6uk9ktuz9ctv
```

Comment:
We'd prefer not to be linking out to a Dropbox link for official integration instructions. Could we get these instructions into an official page on docs.newrelic.com?

Reply:
If we go ahead linking to the dropbox link, then we do not have to maintain the documentation of the particular integration.